01-teams/governance_board.yaml

```diff
@@ -0,0 +1,2 @@
+description: Governance Board
+name: governance_board
```

Comment:
@cobraz you reviewed too fast. I now added a complete description explaining why I'm adding teams to member data files, rather than members to teams data files.